### ISSUE TITLE
encryption issue

### ISSUE THREAD

**Issue by RonnieVisser** [mgp25/Chat-API#1733]
Make sure these boxes are checked before submitting your issue -- thank you!

- [x] You are using PHP > 5.6
- [x] You have installed all extensions and modules listed in [here](https://github.com/mgp25/Chat-API/wiki/Dependencies). PHP Protobuf and Curve25519 are required!
- [x] You have checked if someone has already asked the same issue you have.
- [x] You have read the [wiki](https://github.com/mgp25/Chat-API/wiki).
- [x] Is not a programming question.
- [x] You are using latest Chat API code.

### Error
Hi, I'm having difficulties the getMessages method, sending messages is worked like a charm. The problem I have is that the message is encrypted. I have checked the issues but no solution seems like to work for me. both protobuf and curve are installed. I first spinned up a server using PHP7 but that didn't work with the requirements so I have spinned up a new server using PHP5.6. php -m is now showing both of the extenstion.

I have also tried to remove the axolotl file, but that didn't work either. I even bought a fresh SIM which havent been used before, result stays the same. I don't know what to do next so therefor i'm opening this issue.

below you can find my log file

### Debug log
```xml
tx  <stream:features></stream:features>

tx  <auth user="31617607820" mechanism="WAUTH-2">f96ee811333136313736303738323015403a9f2859bc79b47ed484368c7eb75fef988931343538333732383338</auth>

rx  <from s.whatsapp.net=""></from>

rx  <stream:features></stream:features>

rx  <challenge>cf08ecdab65a99017743811f3d37ce30f825038d</challenge>

tx  <response>78cae5cc03b9d30a81987fb3b8074d40f9dc47de8a98bcdaba5a922795ba23397f9555ec342751fb4265dd9dc4d5ecc7ada92d5c03891ae2896607709c41412f65d067ea639de4bb95681b1c80</response>

rx  <success t="1458372838" props="4" kind="free" status="active" creation="1458311862" expiration="4444444444">af98c48cdde8f6b5faaa1768f60c663d2ced064a</success>

tx  <presence name="Keebo" type="available"></presence>

tx  <iq id="01" xmlns="jabber:iq:privacy" type="get">
tx    <query>
tx      <list name="default"></list>
tx    </query>
tx  </iq>

tx  <iq id="02" xmlns="urn:xmpp:whatsapp:push" type="get" to="s.whatsapp.net">
tx    <config></config>
tx  </iq>

rx  <message from="31646165717@s.whatsapp.net" type="text" id="780952C64849D7DBD904DC0CC6DBEC" offline="3" t="1458337592" notify="Janet">
rx    <enc v="2" type="pkmsg">33122105f55859b3dd0cf3a182f4594f06af8c8c513860f0ceaa0fe24ab57e1efe4f9b5b1a2105aca1a7db7d8e2ea92de42c98a41c81a072478de7b6db170faad0f7bcd196574d22c302330a210597176c172a61aab8c7c76bba0f85c0bf8b465e41719ba98b96f828698c33121510001800229002bf198f0a576473a8d7e9cf3a66a00db292ab8739c3093f76f0a8a77142a9d2f6019bd2c5815eed2913e380b68705d3c6c332f7cf39569683f53f049d5511684e0f74ce2303f339e07bade556dd40bc89031629bbc22e23b109f05a1bd258d582ee5c8943efcb8ef1502ce4f9688871329c6713466ddd3adcc652d18980ad3b7818a59c31c51bebdcda7d26068075668c438ecc43ff32b3ccc60e351bd12683423b310ee0daa6b08e81fdaf953860268d6ec97ea28edd4a710b3cd80b9b4e3fa18d0a1810490fb3d852b293470a5f0d853294b68aa5d3791b0a7c2fdb8db6961752b17ed273390fc831aa60943049b4d30af873c0b705a2c8ebbe358bcdd5433e29158c16f816261ca58747d5eb990c92de5a924b2af027942882e6c98f0230d180a301</enc>
rx  </message>

rx  <receipt from="31657654705@s.whatsapp.net" id="fd18e3be681cb8e9b54bbb" offline="0" t="1458351147"></receipt>

tx  <ack to="31657654705@s.whatsapp.net" class="receipt" id="fd18e3be681cb8e9b54bbb"></ack>

rx  <ib from="s.whatsapp.net">
rx    <offline count="2"></offline>
rx  </ib>

rx  <notification from="s.whatsapp.net" type="encrypt" id="1544669658" t="1458372838">
rx    <count value="0"></count>
rx  </notification>

tx  <ack to="s.whatsapp.net" class="notification" id="1544669658" type="encrypt"></ack>

rx  <presence from="31617607820@s.whatsapp.net"></presence>

rx  <iq from="31617607820@s.whatsapp.net" type="error" id="1">
rx    <error code="404" text="item-not-found"></error>
rx  </iq>

rx  <iq from="s.whatsapp.net" type="error" id="02">
rx    <error code="404" text="item-not-found"></error>
rx  </iq>

tx  <iq id="03" type="set" xmlns="urn:xmpp:whatsapp:push" to="s.whatsapp.net">
tx    <config platform="gcm" id="FalTe_1Wg7c:APA91bPrRPkukqegtIXL9sOnM6XdXJ0Gc76uLAktfJ7HFPXckuVgWBaX7xQoJmcGdh0mQQ_Vfq3s-mRzmhIHh0YTb_zEfGOcFag_17Rj87xdLVf_z_aiO2jTuELkwZRTCfGjzcrbpdlr"></config>
tx  </iq>

rx  <iq from="s.whatsapp.net" type="result" id="03"></iq>
```

**Comment by mgp25** [comment]
The debug log is incomplete, anyway, it seems curve and protobuf extensions are not installed/load correctly. When its loaded correctly messages are decrypted

Regards

**Comment by RonnieVisser** [comment]
You where correct. I updated the CLI php.ini instead of the FPM php.ini file.
 
I now see much more going on but it's still not working. following is the error

-> Decrypted Message: No such signedprekey with id: 11901505 - /home/forge/keebo.nl/releases/20160318212156/vendor/whatsapp/chat-api/src/SqliteAxolotlStore.php - 246Message BDBCAD5FBF49B488C0 could not be decrypted, sending retry. tx  <iq id="30" to="s.whatsapp.net" type="set" xmlns="encrypt">
tx    <identity>15f57daf8b376b7e14106a4c805399ec51732dc5ebe01251a05fd0d9ccc61a4c</identity>

Regarding to some issues I already deleted the axolotl file. Any advice in this?
